native/machine-coding/fitness-tracker/index.js

```diff
@@ -0,0 +1,61 @@
+// JavaScript code for the fitness tracker
+document.addEventListener("DOMContentLoaded", function () {

```

Comment:
wrapping in DOMContentLoaded listener is not needed when script is loaded with defer attirbute